app/server/mergedServerMain.ts

```diff
@@ -58,7 +64,7 @@ export async function main(port: number, serverTypes: ServerType[],
     server.testAddRouter();
   }
 
-  if (options.logToConsole) { server.addLogging(); }
+  if (options?.logToConsole !== false) { server.addLogging(); }
```

Comment:
```suggestion
  if (options.logToConsole !== false) { server.addLogging(); }
```
I think it's ok to leave out the `?`.